core/src/test/java/org/elasticsearch/index/mapper/DynamicMappingDisabledTests.java

```diff
@@ -57,14 +60,8 @@
 
     private static ThreadPool THREAD_POOL;
     private ClusterService clusterService;
-    private Transport transport;
     private TransportService transportService;
-    private IndicesService indicesService;
-    private ShardStateAction shardStateAction;
-    private ActionFilters actionFilters;
-    private IndexNameExpressionResolver indexNameExpressionResolver;
-    private AutoCreateIndex autoCreateIndex;
-    private Settings settings;
+    private TransportIndexAction transportIndexAction;
```

Comment:
we should stop using it :) - maybe have a test utility to index a single item using a bulk action?